libraries/AP_Notify/Buzzer.h

```diff
@@ -63,9 +58,14 @@ class Buzzer: public NotifyDevice
         uint8_t ekf_bad             : 1;    // 1 if ekf position has gone bad
     } _flags;
 
-    uint8_t         _counter;           // reduces 50hz update down to 10hz for internal processing
-    BuzzerPattern   _pattern;           // current pattern
-    uint8_t         _pattern_counter;   // used to time on/off of current patter
-    uint32_t        _arming_buzz_start_ms;  // arming_buzz start time in milliseconds
-    uint8_t         _pin;
+    uint32_t _pattern;           // current pattern
+    uint8_t _pin;
+    uint32_t _pattern_start_time;
+
+    // enforce minumum 100ms interval between patterns:
+    const uint16_t _pattern_start_interval_time_ms = 30*100 + 100;
```

Comment:
That 30 should be 32 (we are only using 30 bit now, but code allows for 32 everywhere).

Reply:
Changed.